docs/antora/modules/ROOT/pages/Library_Dependencies.adoc

```diff
@@ -150,12 +150,26 @@ def compileIvyDeps = Agg(
 )
 ----
 
-It is also possible to use higher version of the same library dependencies already  defined in `ivyDeps`, to ensure you compile against a minimal API version, but actually run with the latest available version.
+It is also possible to use higher version of the same library dependencies already defined in `ivyDeps`, to ensure you compile against a minimal API version, but actually run with the latest available version.
+
+== Detecting transitive dependencies
+
+To render a tree of dependencies (transitive included) and possible upgrades for a given module, you can run `mill myModule.ivyDepsTree`.
+
+You can also compile your module(s) and examine files such as `ivyDeps.json` and `transitiveIvyDeps.json` in your `out` build's folder for a given module. `ivyDepsTree.log` contains file output of the above ivyDepsTree command too.
 
 == Excluding transitive dependencies
 
 You can use the `.exclude` method on a dependency. It accepts  `organization` and `name` tuples, to be excluded.
-Use the special name `*` to match all ``organization``s or ``name``s.
+Use the special name `*` to match all ``organization``s or ``name``s. For example:
+
+.Example
+[source,scala]
+----
+def deps = Agg(
+  ivy"com.lihaoyi::pprint:0.5.3".exclude("com.lihaoyi" -> "fansi_2.12")
```

Comment:
I learnt about this feature by seeing https://github.com/com-lihaoyi/mill/blob/main/scalalib/test/src/ResolveDepsTests.scala#L42

It'd be great to have it documented so it comes up first with internet searches 🚀

Reply:
Question (curiosity, not a suggestion):

There is also `excludeName` and `excludeOrg` - but it sounds like `exclude` could be used in place of either of them (as noted, by using *). 
I guess those other options may be marginally more explicit, for example:

- `exclude(* -> fansi_2.12)` = excludeName(fansi_2.12)
- `exclude(qh.cos.logback -> *)` = excludeOrg(qh.cos.logback)

though it gives a few different ways of doing the right thing. Given that the docs focus on just `exclude`, is it worth marking the others as deprecated?

Reply:
Both are implemented on behalf of `exclude`, so it is only a matter of taste what you want to use. They don't imply a huge maintenance effort either, so unless it confuses people, I don't see the need to deprecate them. Can you mention them in the docs, too?

Reply:
will be in commit ebdd349